server/storage/leveldb_backend.go

```diff
@@ -127,46 +131,46 @@ func (lb *levelDBBackend) SaveRegion(region *metapb.Region) error {
 	return nil
 }
 
-func (lb *levelDBBackend) SaveRegions(regions map[string]*metapb.Region) error {
-	batch := new(leveldb.Batch)
-
-	for key, r := range regions {
-		value, err := proto.Marshal(r)
-		if err != nil {
-			return errs.ErrProtoMarshal.Wrap(err).GenWithStackByCause()
-		}
-		batch.Put([]byte(key), value)
-	}
-
-	if err := lb.Base.(*kv.LeveldbKV).Write(batch, nil); err != nil {
-		return errs.ErrLevelDBWrite.Wrap(err).GenWithStackByCause()
-	}
-	return nil
-}
-
 func (lb *levelDBBackend) DeleteRegion(region *metapb.Region) error {
 	return lb.Remove(endpoint.RegionPath(region.GetId()))
 }
 
-// FlushRegion saves the cache region to region storage.
-func (lb *levelDBBackend) FlushRegion() error {
+// Flush saves the cache region to the underlying storage.
```

Comment:
Why change the name?

Reply:
It should re-implement the `Flush` method of the `endpoint.Region Storage`. I forgot to rename it before.